02.general/20.2FA/docs.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Two-factor authentication'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+***Two-factor authentication***
+
+Two-factor authentication (or 2FA) is an authentication method that requires an additional piece of evidence from the user to log in, on top of the usual login/password pair. Usually for this purpose websites use a code sent by SMS or via a special app.
+
+Now, we believe that 2FA is a strictly positive addition, but we also realize that some users would be irritated by being forced into it. So we made it optional, by default 2FA will be disabled when you register a new personal account on adguard.com. You can enable it manually in the Settings. As for the available options, at the start there will only be one: an authenticator app for Android or iOS — a tried and tested 2FA method (you can use basically any such app, there are plenty).
```

Comment:
We believe that adding a 2FA option is strictly positive, but we also realize that some users would be irritated by being forced into it. So we made it optional for your personal account on adguard.com, and by default 2FA will be disabled when you register a new one. You can enable it manually in the Settings. As for the available options, currently there is only be one: an authenticator app for Android or iOS — a tried and tested 2FA method (you can use basically any such app, there are plenty).